homework/02LessonTwo.js

```diff
@@ -26,50 +26,89 @@ var alwaysTrue = function () {
 // argument1 is equal to argument2
 
 function equals(argument1, argument2) {
+  if (argument1 = argument2)
+    return true;
+    else {
+      return false;
+    }
+
 }
+equals(1, 1);
 
 // Problem 2:
 // lessThanOrEqualTo(parameter1, parameter2) is an empty function
 // return a boolean expression that is true when
 // parameter1 is less than or equal to parameter2
 
 function lessThanOrEqualTo(parameter1, parameter2) {
+    if (parameter1 <= parameter2)
+    return true;
+    else {
+      return false;
+    }
 }
 
+lessThanOrEqualTo(1, 3);
```

Comment:
Not sure if you need to declare this function with (1,3) parameters here... Mocha should do it for you.